### ISSUE TITLE
Provide option to setup snapper if the btrfs filesystem is chosen

### ISSUE THREAD

**Issue by stallonen** [archlinux/archinstall#1808]
While it is not a must have feature,I think it'd be a nice to have snapper config working out of the box.

**Comment by Torxed** [comment]
what additional things is needed other than installing the package?

**Comment by Torxed** [comment]
Humm, previously we had to create `@.snapshots` in order to be compliant with snapper.
I wonder what changed?

Without knowing for sure, it sounds like an issue with snapper not detecting that `@.snapshots` is empty and "ready to be used"? Because we don't create any content in it.

If this is a design choice, absolutely then we can create another question in the btrfs submenu asking if snapshots is desired and set snapper up.

**Comment by chalybesmith** [comment]
I agree this would be a useful feature, since the use for btrfs subvolumes are snapshots in the first place. 
I personally think it would be reasonable (selectable) default option to have snapshots set up by the install-script.

**Comment by Salman7236** [comment]
yes please

**Comment by pocoraven** [comment]
I second this. +1

**Comment by distrair** [comment]
Is there an expected release version where snapper support will be implemented in archinstall?
